src/wallet/rpcwallet.cpp

```diff
@@ -1155,19 +1157,36 @@ UniValue cleanwallettransactions(const UniValue& params, bool fHelp, const CPubK
         // get all locked utxos to relock them later.
         vector<COutPoint> vLockedUTXO;
         pwalletMain->ListLockedCoins(vLockedUTXO);
-        // unlock all coins so that the following call containes all utxos.
+        // unlock all coins so that the following call contains all utxos.
```

Comment:
With using `pwalletMain->mapWallet`, unlocking and then re-locking is no longer necessary.

Reply:
Removed locking/unlocking in a6ac2e7